addOns/pscanrules/src/test/java/org/zaproxy/zap/extension/pscanrules/CacheControlScanRuleUnitTest.java

```diff
@@ -141,8 +142,11 @@ void httpsMissingNoCacheRequest() throws HttpMalformedHeaderException {
         scanHttpResponseReceive(msg);
         // Then
         assertThat(alertsRaised.size(), equalTo(1));
-        assertThat(alertsRaised.get(0).getParam(), equalTo(HttpHeader.CACHE_CONTROL));
-        assertThat(alertsRaised.get(0).getEvidence(), equalTo("no-store, must-revalidate"));
+        Alert alert = alertsRaised.get(0);
+        assertThat(alert.getRisk(), equalTo(Alert.RISK_INFO));
+        assertThat(alert.getConfidence(), equalTo(Alert.CONFIDENCE_LOW));
```

Comment:
All of them could assert the confidence if tomorrow we do tweaks per directive.

Reply:
Sure I can add it to other tests.